### ISSUE TITLE
model.test_on_batch returns total loss instead of average - Keras 2.2.4 

### ISSUE THREAD

**Issue by gowthamkpr** [keras-team/keras#13459]
model.test_on_batch returns total loss instead of average - Keras 2.2.4 

Tensorflow is the issue and it can be fixed using this
